db/migrate/20190322202902_change_approval_state_column_type.rb

```diff
@@ -0,0 +1,10 @@
+class ChangeApprovalStateColumnType < ActiveRecord::Migration[5.2]
+  def up
+    remove_column :approval_requests, :state
+    add_column :approval_requests, :state, :integer, :default => 0
+  end
+
+  def down
+    change_column :approval_requests, :state, :string, :default => 0
```

Comment:
The default should be a string value. "0"

Reply:
agh I had actually changed that but didn't stage it when I committed the rest. It's up now.